core/dutydb/memory.go

```diff
@@ -70,6 +70,8 @@ func (db *MemDB) Shutdown() {
 }
 
 // Store implements core.DutyDB, see its godoc.
+//
+//nolint:gocognit
```

Comment:
cognitive complexity of Store is just right, i don't this can be improved any further